pangeo_forge_recipes/recipes/xarray_zarr.py

```diff
@@ -621,9 +621,25 @@ def store_chunk(
                 zarr_array[zarr_region] = data
 
 
-def finalize_target(target: CacheFSSpecTarget, consolidate_zarr: bool) -> None:
+def finalize_target(
+    target: CacheFSSpecTarget,
+    consolidate_zarr: bool,
+    consolidate_dimension_coordinates: bool = True,
+) -> None:
     if target is None:
         raise ValueError("target has not been set.")
+
+    if consolidate_dimension_coordinates:
+        logger.info("Consolidating dimension coordinate arrays")
+        target_mapper = target.get_mapper()
+        ds = xr.open_zarr(target_mapper)  # Probably a better way to get the dimension coords?
```

Comment:
You could make a set out of the [`_ARRAY_DIMENSIONS`](http://xarray.pydata.org/en/stable/internals/zarr-encoding-spec.html) attribute on each array in the group.

Reply:
Thanks. Done in 1c36e2155c6e86ae633930d541a727713b62efb6.